game_patch/input/mouse.cpp

```diff
@@ -111,6 +112,70 @@ ConsoleCommand2 ms_cmd{
     "ms <value>",
 };
 
+float scope_sensitivity_factor = 1.0f;
+float scanner_sensitivity_factor = 1.0f;
```

Comment:
Seems wordy. Why not `scope_sens` and `scanner_sens`?

Think we should start using `g_` or upper case for globals. @rafalh

Reply:
> Seems wordy. Why not `scope_sens` and `scanner_sens`?

I agree it is wordy, though the impact of that is effectively mitigated by tab complete.

I avoided short hands such as `scanner_sens` and `zoom_sens` (which were the commands I originally had used) based on the standard rafalh has set with most other commands - `reticle_scale`, `spectate_mode_minimal_ui`, etc. I viewed compliance with established convention as more important than being as concise as I otherwise would.